modules/ossm-scoping-service-mesh-with-discoveryselectors.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+// install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-scoping-service-mesh-with-discoveryselectors_{context}"]
+= Scoping the Service Mesh with discovery selectors
+:context: ossm-scoping-service-mesh-with-discoveryselectors
+
+A {SMProductShortName} includes workloads that meet the following criteria:
```

Comment:
"A Service Mesh," "A service mesh," "Service Mesh," or other?

I see most frequently in published docs the common lowercase form when used with an article: https://docs.openshift.com/container-platform/4.17/service_mesh/v2x/ossm-create-mesh.html